### DISCUSSION TITLE
SRT: read size not expected

### DISCUSSION THREAD

**Discussion by NeightDGreat** [Haivision/srt#2584] [Q&A]
Hello,

I have been having issues with an SRT stream and can't for the life of me find any resource on the following log entry:

[info] srt: read size=<376> not expected, data will be dropped (srt://164.52.235.162:9710?adapter=192.168.63.182&minversion=66304&conntimeo=60000&maxbw=20000000&latency=160&mode=caller&passphrase=****************&pbkeylen=16)

The handshake looks good, the RTT is ~35ms, low latency. Not matter what I do, the video is unwatchable, and the audio is basically non-existent. 
What would be a cause for this entry? Is there a resource for finding more information on this?

**Comment by ethouris** [comment]
Need additional info - logs (preferably debug), statistics (need to see lost and dropped packets) and also a recorded pcap from the session, if possible.

This phrase you showed above doesn't seem to come from SRT at all.

**Reply by NeightDGreat** [reply]
Thank you for the quick response. I will work to get that information. The pcap may take some time as this is an occasional source and the next window is late next week. 

That's good to know that this log entry doesn't seem to come from SRT, so it gives me a direction as well.

**Reply by ethouris** [reply]
You haven't even shown what application you are using for it. Anyway, there are many apps you can use, including most directly the `srt-live-transmit` from the SRT package, if it is ok for you to flip it to UDP or a pipeline.
